ref/Microsoft.Build/net/Microsoft.Build.cs

```diff
@@ -1427,6 +1427,7 @@ public sealed partial class ProjectGraph
         public System.Collections.Generic.IReadOnlyCollection<Microsoft.Build.Experimental.Graph.ProjectGraphNode> ProjectNodes { get { throw null; } }
         public System.Collections.Generic.IReadOnlyCollection<Microsoft.Build.Experimental.Graph.ProjectGraphNode> ProjectNodesTopologicallySorted { get { throw null; } }
         public System.Collections.Generic.IReadOnlyDictionary<Microsoft.Build.Experimental.Graph.ProjectGraphNode, System.Collections.Immutable.ImmutableList<string>> GetTargetLists(System.Collections.Generic.ICollection<string> entryProjectTargets) { throw null; }
+        public string ToDot() { throw null; }
```

Comment:
Not sure we'd want this in the public API. It's pretty trivial to create it after the fact. If it helps for testing, it could be made internal?

Reply:
I was thinking it would help have it public in case we decorate the Dot spec with internal information. But maybe this adds even more support to the fact that we need to decorate nodes with tags / metadata like `InnerBuild`, `SpeculativeEdge`, etc